.openpublishing.redirection.json

```diff
@@ -4216,6 +4236,7 @@
         "redirect_url": "/archive/microsoft-edge/legacy/developer",
         "redirect_document_id": false
       },
+// conflict, want to resume using this source_path but for Edge-Chromium article, not EdgeHTML article:
```

Comment:
comment-out the old redirect?  
Want to resume using this source_path but for Edge-Chromium article, not EdgeHTML article.
@captainbrosset 
line 4239 vs. 1365

Reply:
commented out old redir
```
      /* reused this source_path for Edge Chromium article
      {
        "source_path": "./microsoft-edge/webdriver/index.md",
        "redirect_url": "/archive/microsoft-edge/legacy/developer",
        "redirect_document_id": false
      },*/
```
in /webdriver/index.md, in See Also, linked to old:
```
Archived:
* [WebDriver (EdgeHTML)](/archive/microsoft-edge/legacy/developer/webdriver/)
```
@captainbrosset